sdk/identity/azure-identity/test/ut/client_secret_credential_test.cpp

```diff
@@ -14,11 +14,9 @@ using Azure::Identity::Test::_detail::CredentialTestHelper;
 
 TEST(ClientSecretCredential, Regular)
 {
-
   auto const actual = CredentialTestHelper::SimulateTokenRequest(
       [](auto transport) {
         ClientSecretCredentialOptions options;
-        options.AuthorityHost = "https://microsoft.com/";
```

Comment:
What's the rationale for removing setting this option now? Was this redundant before?

Reply:
It is just than now, we don't set it sometimes, so we test the default option as well.